calligraphy/src/main/java/io/github/inflationx/calligraphy3/Calligraphy.java

```diff
@@ -103,6 +107,16 @@ public Calligraphy(CalligraphyConfig calligraphyConfig) {
         this.mAttributeId = new int[]{calligraphyConfig.getAttrId()};
     }
 
+    @NonNull
+    private static String getResourceEntryName(@NonNull View view) {
+        try {
+            return view.getResources().getResourceEntryName(view.getId());
+        } catch (Resources.NotFoundException exception) {
+            Log.w(TAG, "An error occurred when trying to get resource entry name. \nStackTrace:" + exception);
```

Comment:
Let's remove the newline and stacktrace label and just pass the exception as the third method param

Reply:
Done.